ngageoint/opensphere
# feat(chardetng-wasm): pulling chardetng-wasm package into Opensphere

Test Steps:
- OS should build
- Test should run
- Binary files with `chardetng` are detected as plain/text.
